### ISSUE TITLE
Possible Bug: Imported Skillplans Don't Automatically Add Required Skills

### ISSUE THREAD

**Issue by 1adog1** [peterhaneve/evemon#207]
I don't fully recall if this was originally a functionality, but I'm pretty sure it was. When importing a skillplan from a file, the app doesn't automatically add the required skills for everything in the plan. This leads to some strange behavior when trying to modify skills including skills disappearing without warning when trying to change the level they're trained to.

**Comment by peterhaneve** [comment]
I cannot reproduce this issue. Which file type did you use when importing? I tried exporting and re-importing using both EMP and XML formats, and both times dependents skills were added.

**Comment by 1adog1** [comment]
My apologies for not being more specific, here's what I can recall:

_I'll preface this by saying I was working with several XML based characters, each with dozens of dependent EMP Based Skillplans. It's possible there was a complex set of conditions which I can't duplicate now._ 

I first noticed it when I had a character based on a skillplan, which then had a ton of dependent skillplans. Basically I reduced the level of some skills in the skillplan used to create the character, then re-exported character after plan, deleted the old skillplan-based-character, imported the new one, and re-imported all the dependent skillplans. Some skills that relied on the skills I had modified failed to generate those required skills in the new plan, which lead to the behavior I described in the original report. 

The skill that immediately comes to mind is Graviton Physics and the Interdictors Skill; on the initial skillplan I had removed all Science and Graviton Physics training, but when I re-imported an interdictor plan on this new character the Interdictors skill remained intact despite not having Graviton Physics trained, and the Interdictors skill disappeared when I tried to modify its training level. This issue occurred several more times while I was working with these skillplans, including with a hictor skillplan again related to the remove of Graviton Physics. 

I can't reproduce the problem now even under the above conditions though, so it's possible that it was some sort of caching error, but I'm pretty sure restarting the client was one of the first things I tried. The original files which produced the error have gone through several iterations since it occurred, so I don't think it's possible for me to perfectly re-create the original conditions.
